### PR TITLE
Add op define extra for norm and frobenius norm op.

### PR DESCRIPTION
**Pull request by ZHUI**
### PR types

Others
### PR changes

OPs
### Describe

Add op define extra for norm and frobenius norm op.

### CODE REVIEW

File: paddle/fluid/operators/norm_op.cc [hunk 1]
```diff
@@ -35,7 +35,13 @@ class NormOpMaker : public framework::OpProtoAndCheckerMaker {
...
               "(Tensor) A tensor saved the `sqrt(sum(x) + epsion)` will "
               "be used in backward kernel.")
-        .AsIntermediate();
+        .AsIntermediate()
+        .AsExtra();
+    AddAttr<bool>("is_test",
+                  "(bool, default false) Set to true for inference only, false "
+                  "for training.")
+        .SetDefault(false)
+        .AsExtra();
```
Comment (winter-wang) [inline paddle/fluid/operators/norm_op.cc]: 这个is_test不能被标记，不然导出时裁了，再次导入通过exe.run（）执行时会出问题

**Review by winter-wang — CHANGES_REQUESTED** [review]

File: paddle/fluid/operators/norm_op.cc (hunk 1 above)
Comment (ZHUI) [reply paddle/fluid/operators/norm_op.cc]: deleted

**Review by winter-wang — APPROVED** [review]
LGTM

**Review by zhangting2020 — APPROVED** [review]
LGTM for skip_check_grad_ci

**Pull request merged**